src/lang/lwc_jaJP.properties

```diff
@@ -619,4 +619,4 @@ lwc.time.second=秒
 lwc.time.seconds=秒
 lwc.time.early=現在
 lwc.time.unknown=フメイ
-lwc.time.past=%time% 経過
+lwc.time.past=%time% 経過
```

Comment:
There still appears to be a diff here. Please see "Files changed" after you fix this to verify that there are no further changes so I can merge this.